src/test/regress/expected/bfv_planner.out

```diff
@@ -880,13 +880,13 @@ from
                                  Hash Cond: (t1_12146.b = t3_12146.b)
                                  ->  Seq Scan on t1_12146  (cost=10000000000.00..10000000112.00 rows=5000 width=8)
                                  ->  Hash  (cost=10000000003.40..10000000003.40 rows=7 width=8)
-                                       ->  Broadcast Motion 3:2  (slice1; segments: 3)  (cost=10000000000.00..10000000003.40 rows=7 width=8)
+                                       ->  Broadcast Motion 3:3  (slice1; segments: 3)  (cost=10000000000.00..10000000003.40 rows=7 width=8)
```

Comment:
I think, due to the comments in the query, this test was broken inappropriately.

Reply:
The number of motion segments is calculated in [FillSliceGangInfo](https://github.com/arenadata/gpdb/blob/7d480f8cc20d8632eddaa75007b1105d5dee3eff/src/backend/executor/execMain.c#L5055). Real gang size is calculated for direct dispatch nodes only, but this one isn't considered as one. As far as I understand, direct dispatch is achievable if a table is joined with some constexpr

Reply:
so, comments in the query are wrong?